src/main/scala/za/co/absa/hyperdrive/trigger/scheduler/utilities/Configs.scala

```diff
@@ -29,13 +29,14 @@ private object Configs {
 
   def getMapFromConf(propertyName: String): Map[String, String] = {
     Try {
-      val list = Configs.conf.getObjectList(propertyName).asScala
-      (for {
-        item: ConfigObject <- list
-        entry <- item.entrySet().asScala
-        key = entry.getKey
-        value = entry.getValue.unwrapped().toString
-      } yield (key, value)).toMap
+      def getKeys(path: String): Seq[String] = {
```

Comment:
Have you seen this [API DOC](https://lightbend.github.io/config/latest/api/com/typesafe/config/ConfigObject.html#unwrapped--)

Reply:
You are right. I have created issue #43 for that and it has to be properly handled.